esrally/client/asynchronous.py

```diff
@@ -202,22 +202,38 @@ def response(self, path):
 class RallyAiohttpHttpNode(elastic_transport.AiohttpHttpNode):
     def __init__(self, config):
         super().__init__(config)
-
-        self._loop = asyncio.get_running_loop()
-        self._limit = None
-
-        client_options = config._extras.get("_rally_client_options")
-        if client_options:
-            self._trace_configs = client_options.get("trace_config")
-            self._enable_cleanup_closed = client_options.get("enable_cleanup_closed")
-
-        static_responses = client_options.get("static_responses")
-        self.use_static_responses = static_responses is not None
-
-        if self.use_static_responses:
+        self._loop = None
+        self._trace_configs = None
+        self._enable_cleanup_closed = None
+        self._static_responses = None
+
+    @property
+    def trace_configs(self):
+        return self._trace_configs
+
+    @trace_configs.setter
+    def trace_configs(self, trace_configs):
+        self._trace_configs = [trace_configs]
```

Comment:
Making this a list in the setter seems very confusing to me, can we do that in create_async instead? And then we can get away with the getter/setter.